dashboards/ds-solarwinds-webapi/readme.md

```diff
@@ -0,0 +1,12 @@
+# SolarWinds Orion Nodes `API`
+
+This dashboard gives an overview of nodes alongside summary of heatlh from SolarWinds Orion using the powerful Web API and dashboard sharing capabilities of SquaredUp. 
+
+## Requirements
+
+- Copy the images repository on the SquaredUp IIS folder (or individually if the images repository already exists)
```

Comment:
Maybe mention that the folder `images` needs to exist specifically in the IIS root i.e. `C:\inetpub\wwwroot` and the structure of `\health-icons\` and `\logos\` should be maintained (or the dashboard JSON updated to suit).

Reply:
Do we want to detail the provider side of this at all? We skipped this during our discussion on Friday.

Reply:
Does it make sense to have the images folder as its own folder with a readme we can link to?